draft-ietf-dprive-dnsoquic.md

```diff
@@ -204,7 +204,8 @@ to QUIC streams in {{stream-mapping-and-usage}}.
 
 Using QUIC might allow a protocol to disguise its purpose from devices on the
 network path using encryption and traffic analysis resistance techniques like
-padding. This specification does not include any measures that are designed to
+padding, traffic pacing, and traffic shaping. This specification does not
+include any measures that are designed to
 avoid such classification. Consequently, firewalls and other middleboxes might
 be able to distinguish DoQ from other protocols that use QUIC, like HTTP, and
 apply different treatment.
```

Comment:
There is a related comment in the review by @martinduke :
```
...but then Sec 6.4 has a detailed, normative discussion of how to use padding to
avoid classification. I suggest you delete or edit the bit in 4.3.
```
To Martin's point, the padding specified in 6.4 is attempting to hide which names and resource records are queried, but not trying to hide that this is DNS over QUIC rather than, for example, HTTP3 or Media over QUIC. Any suggestion on how to phrase that properly?

Reply:
Let's address Martin's point in a separate PR.